ci-operator/step-registry/openshift/kubernetes/e2e/nvidia-dra-partitionable/openshift-kubernetes-e2e-nvidia-dra-partitionable-commands.sh

```
@@ -0,0 +1,55 @@
+set -o pipefail
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+export ARTIFACTS="${ARTIFACT_DIR}"
+
+readonly GPU_LABEL="nvidia.com/gpu.present=true"
+readonly DRA_DEVICECLASS="nvidia.com/gpu"
+
+echo "Test Focus: ${DRA_TEST_FOCUS}"
+
+# Verify GPU nodes
+readonly GPU_NODES=$(oc get nodes -l "${GPU_LABEL}" -o name 2>/dev/null | wc -l)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)